frontend/src/components/notifications/NotificationDropdown.jsx

```
@@ -0,0 +1,109 @@
+  
+      {notifications.length === 0 ? (
+        <p className="p-4 text-gray-500">No notifications</p>
+      ) : (
+        notifications.map((notif) => (
+          <div
+            key={notif.id}
+            onClick={() => handleNotificationClick(notif.post_id, notif.id)}
+            className={`flex items-start gap-3 p-3 border-b cursor-pointer transition-all relative hover:bg-gray-50 ${
+              notif.is_read ? 'bg-white text-gray-600' : 'bg-blue-50 text-gray-800'
+            }`}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)